level-1/평균-구하기.js

```diff
@@ -30,3 +30,12 @@ function solution(arr) {
   });
   return answer / arr.length;
 }
+
+//정답 5 - yongchanson
+function solution(arr) {
+  let sum = 0;
+  for (var i = 0; i < arr.length; i++) {
+    sum += arr[i];
+  }
+  return sum / i;
```

Comment:
🌱 웬만하면 반영해주세요. 🌱

var의 호이스팅 덕분에 잘 작동하는 것 같지만, 권장되지 않습니다!
```suggestion
  return sum / arr.length;
```

Reply:
호이스팅 이용하는게 제 의도였는데
권장되지 않는 사항이라면 빼겠습니다!